contlcycle/contlcycle.pb.go

```diff
@@ -2,18 +2,16 @@
 // source: proto/contlcycle/contlcycle.proto
 
 /*
-Package contlcycle is a generated protocol buffer package.
+	Package contlcycle is a generated protocol buffer package.
```

Comment:
Hmm.. is there a reason why these unrelated files have changed?

Reply:
I'm not sure why re-generating the code changed that, I guess I'm not using the same versions as before. But I can't see in the readme what version I should use.